opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -249,13 +252,51 @@ namespace Opm {
     }
 
 
+    template<typename FluidSystem, typename BlackoilIndices, typename ElementContext, typename MaterialLaw>
+    template <typename Simulator>
+    void
+    StandardWellsDense<FluidSystem, BlackoilIndices, ElementContext, MaterialLaw>::
+    getMobility(const Simulator& ebosSimulator, const int perf, const int cell_idx, std::vector<EvalWell>& mob) const
+    {
+
+        const int np = wells().number_of_phases;
+        assert (mob.size() == np);
+        const auto& intQuants = *(ebosSimulator.model().cachedIntensiveQuantities(cell_idx, /*timeIdx=*/0));
```

Comment:
I think that having to resort to direct IQ cache usage is not very elegant here, but I agree with @blattms that it should not cause trouble. it's probably one of the cases where it must be accepted with a fist in the pocket, at least until the well model gets some refactoring love...